### DISCUSSION TITLE
Laravel 9 Near Future Support

### DISCUSSION THREAD

**Discussion by Loydtafireyi** [rappasoft/laravel-livewire-tables#810] [Feature Requests]
Do you see Laravel Livewire tables supporting Laravel 9 in the near future?

**Comment by dgillier** [comment]
I'm using Laravel 9.17 and the package is running fine on it ;)

**Reply by Loydtafireyi** [reply]
@dgillier thanks for the reply is it also working in production with no issues?

**Reply by dgillier** [reply]
Don't know yet ;)

**Reply by CovertError** [reply]
works pretty well with me
